### PR TITLE
README.es.md

### PR DESCRIPTION
**Pull request by UlisesTJ**
Any suggestions? I'll gladly made as many commits as necessary! I'd love to promote this project to the spanish-speaking community I hope you guys liked my contribution and be able to merge this!

### CODE REVIEW

**Comment by UlisesTJ** [comment]
@geerlingguy @ashabed

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+[Drupal VM](https://www.drupalvm.com/) Es un VM para Drupal, construido con Ansible.
+
+Drupal VM hace que la construcción de entornos de desarrollo Drupal sea rápido y fácil, y presenta a los programadores al maravilloso mundo del programador Drupal en máquinas virtuales o contenedores Docker (en vez del viejo programador MAMP/WAMP-based).
+
+Instalará los siguientes en un Ubuntu 16.04 (predeterminado) linux VM:
+
+  - Apache 2.4.x (or Nginx)
```
Comment (geerlingguy) [inline README.es.md]: Can you also translate the contents of this list (e.g. `or` to `o`, `for accessing databases directly`, etc.)?

**Comment by UlisesTJ** [comment]
@geerlingguy done! check it out! any other suggestion?

**Comment by UlisesTJ** [comment]
Hello @geerlingguy Have you taken a look at the translation? :+1: let me know!

**Comment by UlisesTJ** [comment]
Hi! @geerlingguy are you going to merge this? Let me know :+1:

**Comment by UlisesTJ** [comment]
Hi! @geerlingguy Let me know if there are any changes left before merging! :+1:  I'm available to make more contributions (translations) if the PR is over a thousand words! Let me know :1st_place_medal:

**Pull request closed without merging**